tools/featurecounts/README.rst

```diff
@@ -0,0 +1,55 @@
+FeatureCounts wrapper for Galaxy
```

Comment:
Als we al onze tools gaan verhuizen naar deze repo is dat wel handig, anders raakt de geschiedenis verdwenen. We kunnen ze wel gelijk in ./deprecated zetten als je dat liever hebt.